### PR TITLE
Fixed #31211

### LINKED ISSUE #31211: [Vault] Template term is not translatable
Unable to create translation for [highlighted term](https://github.com/magento/magento2/blob/9544fb243d5848a497d4ea7b88e08609376ac39e/app/code/Magento/Vault/view/frontend/templates/customer_account/credit_card.phtml#L46-L48)

### Preconditions
1. Magento 2.3+
2. Payment method with vault support and using [native template](https://github.com/magento/magento2/blob/9544fb243d5848a497d4ea7b88e08609376ac39e/app/code/Magento/Vault/view/frontend/templates/customer_account/credit_card.phtml).

### Steps to reproduce
1. Store uses any language other than en_US.
2. Use a payment method that allows you to save a card. (This method should consume the native template for managing stored payment methods).
3. Go on My Account -> Stored Payment Methods, click on delete

### Expected resul

The content is not translatable.

### Actual result

In a translation file for any language you can translate the term.

### Possible solution

Presented on the pull

Please provide [Severity](https://devdocs.magento.com/guides/v2.3/contributor-guide/contributing.html#backlog) assessment for the Issue as Reporter. This information will help during Confirmation and Issue triage processes.

- [x] Severity: **S0** _- Affects critical data or functionality and leaves users without workaround._
- [ ] Severity: **S1** _- Affects critical data or functionality and forces users to employ a workaround._
- [ ] Severity: **S2** _- Affects non-critical data or functionality and forces users to employ a workaround._
- [ ] Severity: **S3** _- Affects non-critical data or functionality and does not force users to employ a workaround._
- [ ] Severity: **S4** _- Affects aesthetics, professional look and feel, “quality” or “usability”._

### PR DESCRIPTION
**Pull request by elisei**
### Description
It makes it possible to translate the term in the template to manage stored payment methods.

### Fixed Issues (if relevant)
1. Fixes magento/magento2#31211

### Manual testing scenarios (*)
1. Not Apply

### Contribution checklist (*)
 - [x ] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by ptylek** [comment]
@magento run all tests

File: app/code/Magento/Vault/view/frontend/templates/customer_account/credit_card.phtml [hunk 1]
```diff
@@ -43,9 +43,7 @@ $ccNumberView = $block->escapeHtml($block->getNumberLast4Digits());
                                         "modalClass": "my-credit-cards-popup",
                                         "toggleEvent": "click",
                                         "title": "<?= $block->escapeHtml(__('Delete')) ?>",
-                                        "content": "Are you sure you want to delete this card: <?=
-                                            /* @noEscape */ $ccNumberView
-                                        ?>?"
+                                        "content": "<?= $block->escapeHtml(__('Are you sure you want to delete this card: %1?', /* @noEscape */ $ccNumberView)) ?>"
```
Comment (ihor-sviziev) [inline app/code/Magento/Vault/view/frontend/templates/customer_account/credit_card.phtml]: no escape here is not really needed, please remove it
```suggestion
                                        "content": "<?= $block->escapeHtml(__('Are you sure you want to delete this card: %1?', $ccNumberView)) ?>"
```

File: app/code/Magento/Vault/i18n/en_US.csv [hunk 2]
```diff
@@ -13,3 +13,4 @@ Actions,Actions
 Delete,Delete
 "PayPal Account","PayPal Account"
 Cancel,Cancel
+"Are you sure you want to delete this card: %1?","Are you sure you want to delete this card: %1?"
```
Comment (ihor-sviziev) [inline app/code/Magento/Vault/i18n/en_US.csv]: please add the new line at the end of the file, as it was before
```suggestion
"Are you sure you want to delete this card: %1?","Are you sure you want to delete this card: %1?"

```

**Review by ihor-sviziev — CHANGES_REQUESTED** [review]

File: app/code/Magento/Vault/i18n/en_US.csv (hunk 2 above)
Comment (elisei) [reply app/code/Magento/Vault/i18n/en_US.csv]: thanks, it's done!

**Comment by elisei** [comment]
@magento run all tests

**Comment by ihor-sviziev** [comment]
@magento run Functional Tests EE

**Review by ihor-sviziev — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @ihor-sviziev, thank you for the review.
[ENGCOM-8553](https://jira.corp.magento.com/browse/ENGCOM-8553) has been created to process this Pull Request

**Comment by magento-engcom-team** [comment]
@elisei thank you for contributing. Please accept Community Contributors team invitation [here](https://github.com/orgs/magento/invitation) to gain extended permissions for this repository.

**Comment by engcom-Alfa** [comment]
@magento give me test instance

**Comment by engcom-Alfa** [comment]
:heavy_check_mark: QA Passed

**Manual testing scenario:**

1. Store uses any language other than en_US.
2. Use a payment method that allows you to save a card. (This method should consume the native template for managing stored payment methods).
3. Go on My Account -> Stored Payment Methods, click on delete

**Before:** :heavy_multiplication_x: The content is not translatable.

Screenshot from 2020-12-18 14-58-04

**After:** :heavy_check_mark: Template term was successfully translated

**Pull request merged**